ext_emconf.php

```diff
@@ -8,12 +8,11 @@
     'uploadfolder' => 0,
     'createDirs' => '',
     'clearCacheOnLoad' => 0,
-    'version' => '8.0.2',
+    'version' => '9.5.0',
     'constraints' => [
         'depends' => [
-            'typo3' => '8.7.0-8.7.99',
-            'sv' => '8.7.0-8.7.99',
```

Comment:
I guess you removed it due to not being available in v9 anymore. I still suggest adding it to the suggests section, in order to have a correct dependency chain (loading order) in v8.
It is of course not critical as core extensions are loaded first anyways.

Reply:
Also not critical because openid can be loaded in any order. It does not use anything from other extensions during TYPO3 initialisation.

Reply:
I think we can just ignore that. Otherwise it may confuse people when installing the ext in the EM.